package.json

```
@@ -24,23 +24,21 @@
     ]
   },
   "devDependencies": {
+    "@skyscanner/eslint-config-skyscanner": "^11.1.0",
     "babel-eslint": "^10.0.1",
-    "ensure-node-env": "^1.0.8",
-    "eslint": "^5.15.3",
-    "eslint-config-airbnb": "^17.1.0",
-    "eslint-config-skyscanner": "^4.0.0",
+    "eslint": "^7.32.0",
     "eslint-plugin-backpack": "^0.2.3",
-    "eslint-plugin-import": "^2.2.0",
-    "eslint-plugin-jsx-a11y": "^6.1.1",
-    "eslint-plugin-react": "^7.11.1",
+    "eslint-plugin-import": "^2.25.2",
+    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-react": "^7.29.4",
     "husky": "^1.0.1",
-    "jest": "^24.5.0",
+    "jest": "^27.5.1",
     "jsdoc": "^3.4.3",
     "jsdoc-to-markdown": "^4.0.1",
     "lint-staged": "^8.1.5"
   },
   "peerDependencies": {
-    "eslint": "^3.0.0 || ^4.0.0 || ^5.0.0"
+    "eslint": "^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0 || ^7.0.0"
```

```suggestion
    "eslint": "^5.0.0 || ^6.0.0 || ^7.0.0"
```
For version 5, it has been 3 years ago when it was first released, for security concern, I think version 3 and version 4 should be abandoned.

---

That sounds reasonable but we would need to consider it a breaking change if we do that.

I suggest we drop 3.x & 4.x as part of the next major version release which will be happening when Sophia makes her changes to recommend using date-fns